concent_api/api-integration-test.py

```diff
@@ -9,16 +9,19 @@
 
 from golem_messages         import dump
 from golem_messages         import load
+from golem_messages.message import AckReportComputedTask
+from golem_messages.message import ComputeTaskDef
+from golem_messages.message import ForceReportComputedTask
 from golem_messages.message import Message
-from golem_messages.message import MessageAckReportComputedTask
-from golem_messages.message import MessageForceReportComputedTask
-from golem_messages.message import MessageTaskToCompute
+from golem_messages.message import TaskToCompute
 
 from utils.testing_helpers  import generate_ecc_key_pair
 
 import requests
 
-(CONCENT_PRIVATE_KEY,   CONCENT_PUBLIC_KEY)   = generate_ecc_key_pair()
+CONCENT_PUBLIC_KEY  = b'\xf3\x97\x19\xcdX\xda\x86tiP\x1c&\xd39M\x9e\xa4\xddb\x89\xb5,]O\xd5cR\x84\xb85\xed\xc9\xa17e,\xb2s\xeb\n1\xcaN.l\xba\xc3\xb7\xc2\xba\xff\xabN\xde\xb3\x0b\xa6l\xbf6o\x81\xe0;'
+CONCENT_PRIVATE_KEY = b'l\xcdh\x19\xeb$>\xbcG\xa1\xc7v\xe8\xd7o\x0c\xbf\x0e\x0fM\x89lw\x1e\xd7K\xd6Hnv$\xa2'
+
```

Comment:
Keys are hard-coded there because I cannot get them from settings. This script is out of django and I can't use development settings and they cant be generated because when we use script for testing cluster we need exactly the same keys are as on the cluster